### ISSUE TITLE
support for settings.n in ollama

### ISSUE THREAD

**Issue by cognitivetech** [cosmicoptima/loom#31]
https://github.com/ollama/ollama/issues/7518

trying to figure out why I get only 1 generation from ollama. 

I suspect this is due to constraints of its api, but noting here for future discussion

perhaps related to this same issue with llama.cpp server? https://github.com/cosmicoptima/loom/pull/25

**Comment by deltanym** [comment]
yup llama.cpp and ollama do not have n. we need to take it up with them
